scripts/release_notes/commitlist.py

```diff
@@ -74,13 +80,20 @@ def keywordInFile(file, keywords):
         return False
 
     @staticmethod
-    def categorize(commit_hash, title):
+    def gen_commit(commit_hash):
```

Comment:
This line calls `CommitList.categorize` with 2 arguments, so it will result in an error currently: https://github.com/pytorch/pytorch/blame/793621984bc192f2e290dfcaa4f9e035f8b81cb8/scripts/release_notes/commitlist.py#L266